src/Form/Field.php

```diff
@@ -316,11 +316,14 @@ protected function formatId($column)
      */
     protected function formatLabel($arguments = []): string
     {
-        $column = is_array($this->column) ? current($this->column) : $this->column;
+        if ($label) {
+            return $label;
+        }
 
-        $label = $arguments[0] ?? ucfirst($column);
+        $column = is_array($this->column) ? current($this->column) : $this->column;
+        $label = ucfirst($column);
 
-        return str_replace(['.', '_', '->'], ' ', $label);
+        return __(str_replace(['.', '_', '->'], ' ', $label));
```

Comment:
Please leave the translation handling to the usercode, outside of laravel-admin. There it's easier to do and easier to override.

Reply:
Ok, I made all formatLabel methods work the same way - I removed translations.